staff/jtibau/exercises-form/form.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML5 Form Exercise</title>
+</head>
+<body>
+    <h1>HTML5 Form Exercise</h1>
+    <h2><b>A Simple Form</b></h2>
+    <h4><i>Form Fudamentals</i></h4>
+    <form action="/send.html" method="post">
```

Comment:
Good w/ the `action` but an `.html` will never be able to process the data sent by a form. Most of the cases you'll use simply `action="/send"`